avalanche/evaluation/metric_utils.py

```diff
@@ -216,10 +296,11 @@ def get_metric_name(metric: 'PluginMetric',
 
 
 __all__ = [
-    'default_cm_image_creator',
-    'get_task_label',
-    'phase_and_task',
-    'get_metric_name',
-    'stream_type',
-    'bytes2human'
+    "default_cm_image_creator",
+    "get_task_label",
+    "phase_and_task",
+    "get_metric_name",
+    "stream_type",
+    "bytes2human",
+    "repartition_pie_chart_image_creator",
```

Comment:
I think you should add also the other 2 newly created functions in the `__all__` since they are imported later on (`default_history_repartition_image_creator` and `repartition_bar_chart_image_creator`)